src/plone/app/theming/configure.zcml

```diff
@@ -11,6 +11,7 @@
     <include package="plone.resourceeditor" />
     <include package="plone.transformchain" />
     <include package="plone.app.registry" />
+    <include package="plone.restapi" />
```

Comment:
I have some doubt about this line.
Is this change important only for the tests?
In that case I would handle that in the test layer.

And probably add `plone.restapi` as an extra requirement in the `setup.py` for the `[test]` feature.